.github/workflows/publish-to-npm.yml

```diff
@@ -1,9 +1,10 @@
 name: publish-to-npm
 on:
-    pull_request:
-        types: [closed]
-        branches:
-            - main
```

Comment:
Hey @revanth0212, do we want to publish on every push? or reinstate the original condition to just publish when the pr to main  is closed

Reply:
I pushed a change to remove this 5 mins ago https://github.com/adobe/aio-cli-plugin-api-mesh/pull/66/commits/06a3cc93ffeac1fe1059f4c714ea62a01fd908b9

It was a temp change that shouldnt go to PROD